pyvista/plotting/plotting.py

```diff
@@ -3025,7 +3025,8 @@ def deep_clean(self):
         self.textActor = None
 
     def add_text(self, text, position='upper_left', font_size=18, color=None,
-                 font=None, shadow=False, name=None, viewport=False):
+                 font=None, shadow=False, name=None, viewport=False, *,
+                 render=True):
```

Comment:
Huh. Just learned about `*,` in [PEP 3102](https://www.python.org/dev/peps/pep-3102/)

Reply:
@banesullivan as of 3.8 we've got even funkier, [positional-only params](https://www.python.org/dev/peps/pep-0570/#specification).

Reply:
IMO, keyword arguments should always be used as such and we should avoid passing positional arguments to those... makes me think we need to do this in more place

Reply:
Ah! Funky indeed, but useful